### PR TITLE
Highlight chunks overlap issues in the new FAQ section

### LINKED ISSUE #579: [Docs RFC] Update caggs docs to explain different retention policies
# Describe change in content, appearance, or functionality

> If I have a table/hypertable with a retention policy of a week, but have the continuous aggregations retention policy being a month... is that going to be OK?

> Yes. It mean your raw data will be around for a week while continuous aggregates will be around for a month.

# Subject matter expert (SME)

@jonatas 

# Deadline

[When does this need to be addressed]

# Any further info

Link to Community Slack thread: https://timescaledb.slack.com/archives/C4GT3N90X/p1635806655175900

### PR DESCRIPTION
**Pull request by jonatas**
# Description

Add FAQ and highlight small fragment related to chunks time overlap between retention policies.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #579.

### CODE REVIEW

**Comment by Loquacity** [comment]
Instead of adding this here as an FAQ, let's put it in `timescaledb/how-to-guides/continuous-aggregates/troubleshooting.md` and frame it as troubleshooting.

File: timescaledb/how-to-guides/continuous-aggregates/troubleshooting.md [hunk 1]
```diff
@@ -77,3 +77,12 @@ dev=# select * FROM (select time_bucket_gapfill(4, time,-5,13), locf(avg(v)::int
...
 ```
+
+## Compression policies
+
+
+If I have a hypertable with a retention policy of a week, but have the continuous
+aggregations retention policy being a month. Is that going to be OK?
+
+Yes. It means the raw data will be around for a week while continuous
+aggregates will be around for a month.
```
Comment (Loquacity) [inline timescaledb/how-to-guides/continuous-aggregates/troubleshooting.md:88]: You didn't read the instructions at the top of the page 😉 

```suggestion
## Compression policies
If you have hypertables using a different retention policy to your 
continuous aggregates, the retention policies are applied separately. 
The retention policy on a hypertable determines how long the raw 
data is kept for. The retention policy on a continuous aggregate 
determines how long the continuous aggregate is kept for. For 
example, if you have a hypertable with a retention policy of a 
week, but a continuous aggregate with a retention policy of a 
month, the raw data is kept for a week, and the continuous
aggregate is kept for a month.
```

Also, move it up the page to put it in alphabetical order ☝️

File: timescaledb/how-to-guides/data-retention/data-retention-with-continuous-aggregates.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ that any chunks that are dropped in the region still being refreshed by the
 continuous aggregate causes the chunk data to disappear from the aggregate as
 well. If the intent is to keep the aggregate while dropping the underlying data,
 the interval being dropped should not overlap with the offsets for the continuous
-aggregate. Learn more [here](#FAQ)
+aggregate. Learn more [here][troubleshooting].
```
Comment (Loquacity) [inline timescaledb/how-to-guides/data-retention/data-retention-with-continuous-aggregates.md:11]:
```suggestion
aggregate. For more information, see the [troubleshooting section][troubleshooting].
```

File: timescaledb/how-to-guides/data-retention/data-retention-with-continuous-aggregates.md [hunk 3]
```diff
@@ -46,14 +46,7 @@ this to also drop the aggregate data after 600 days:
...
-1. If I have a hypertable with a retention policy of a week, but have the continuous
-aggregations retention policy being a month. Is that going to be OK?
-
-Answer: Yes. It means the raw data will be around for a week while continuous
-aggregates will be around for a month.
-
 [drop_chunks]: /api/:currentVersion:/hypertable/drop_chunks
 [add_retention_policy]: /api/:currentVersion:/data-retention/add_retention_policy
 [continuous_aggregates]: /how-to-guides/continuous-aggregates
+[troubleshooting]: /how-to-guides/continuous-aggregates/troubleshooting.md
```
Comment (Loquacity) [inline timescaledb/how-to-guides/data-retention/data-retention-with-continuous-aggregates.md:52]:
```suggestion
[troubleshooting]: /how-to-guides/continuous-aggregates/troubleshooting/
```

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: timescaledb/how-to-guides/continuous-aggregates/troubleshooting.md (hunk 1 above)
Comment (jonatas) [reply timescaledb/how-to-guides/continuous-aggregates/troubleshooting.md:88]: oush, my apologies! adopting the suggestion 🙇

**Review by Loquacity — APPROVED** [review]

**Pull request merged**